app/konnect/updates.md

```diff
@@ -9,8 +9,12 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
-## April 2023
+## May 2023
+
+**Gateway 3.3 Support**
+: Konnect now supports the latest Gateway release version of 3.3 including all plugins.
```

Comment:
```suggestion
**{{site.base_gateway}} 3.3 support*
: {{site.konnect_short_name}} now supports {{site.base_gateway}} 3.3.x, including all plugins. See the [Gateway changelog](https://docs.konghq.com/gateway/changelog/#3300) for details on all the updates and new features.
```

Our of curiousity, why are we specifically calling out plugins? Also saying "including all plugins" could be misconstrued as all plugins available in Gateway 3.3, which would be wrong - there will always be a bunch of plugins unsupported in Konnect.

Reply:
@lena-larionova good question/feedback - let me review w/ Smriti

Reply:
Thats a good feedback. I like Amy's suggestion about rephrasing as "all Konnect-compatible plugins" with a link to https://docs.konghq.com/konnect/compatibility/